### PR TITLE
cli: add unsafe sub command and hide it from the default help info

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
add unsafe sub command and hide it from the default help info

```
➜  ticdc git:(add_unsafe_cmd) ✗ ./bin/cdc cli --help
Manage replication task and TiCDC cluster

Usage:
  cdc cli [flags]
  cdc cli [command]

Available Commands:
  capture     Manage capture (capture is a CDC server instance)
  changefeed  Manage changefeed (changefeed is a replication task)
  processor   Manage processor (processor is a sub replication task running on a specified capture)
  tso         Manage tso

Flags:
      --ca string          CA certificate path for TLS connection
      --cert string        Certificate path for TLS connection
  -h, --help               help for cli
  -i, --interact           Run cdc cli with readline
      --key string         Private key path for TLS connection
      --log-level string   log level (etc: debug|info|warn|error) (default "warn")
      --pd string          PD address (default "http://127.0.0.1:2379")

Use "cdc cli [command] --help" for more information about a command.

➜  ticdc git:(add_unsafe_cmd) ✗ ./bin/cdc cli unsafe --help
Usage:
  cdc cli unsafe [command]

Available Commands:
  delete-service-gc-safepoint Delete CDC service GC safepoint in PD, confirm that you know what this command will do and use it at your own risk
  reset                       Reset the status of the TiCDC cluster, delete all meta data in etcd, confirm that you know what this command will do and use it at your own risk

Flags:
  -h, --help         help for unsafe
      --no-confirm   Don't ask user whether to confirm executing meta command

Global Flags:
      --ca string          CA certificate path for TLS connection
      --cert string        Certificate path for TLS connection
  -i, --interact           Run cdc cli with readline
      --key string         Private key path for TLS connection
      --log-level string   log level (etc: debug|info|warn|error) (default "warn")
      --pd string          PD address (default "http://127.0.0.1:2379")

Use "cdc cli unsafe [command] --help" for more information about a command.


```

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cmd/client_unsafe.go [hunk 1]
```diff
@@ -22,6 +22,18 @@ import (
...
+	}
+	command.AddCommand(
+		newMetadataCommand(),
+	)
+	return command
+}
+
 func newMetadataCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "meta",
```
Comment (overvenus) [inline cmd/client_unsafe.go:39]: What about renaming 'meta' to 'unsafe'?

**Comment by amyangfei** [comment]
What's going on with this PR @leoppro

**Pull request merged**